packages/asset-swapper/contracts/src/AaveV2Sampler.sol

```diff
@@ -0,0 +1,94 @@
+// SPDX-License-Identifier: Apache-2.0
+/*
+
+  Copyright 2021 ZeroEx Intl.
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+
+*/
+
+pragma solidity ^0.6;
+pragma experimental ABIEncoderV2;
+
+import "@0x/contracts-zero-ex/contracts/src/transformers/bridges/mixins/MixinAaveV2.sol";
+import "./SwapRevertSampler.sol";
+
+contract AaveV2Sampler is
+    MixinAaveV2,
+    SwapRevertSampler
+{
+
+    function sampleSwapFromAaveV2(
+        address sellToken,
+        address buyToken,
+        bytes memory bridgeData,
+        uint256 takerTokenAmount
+    )
+        external
+        returns (uint256)
+    {
+        return _tradeAaveV2(
```

Comment:
@kimpers just curious why using the _trade function instead of just returning the same takerTokenAmount? (1=1 for aTokens)

Reply:
oh this is using the sample with SwapRevert method

Reply:
This branch doesn't actually work, I should close this PR. I was experimenting with SwapRevert but ran into some issues. #293 is the PR ready for review. Sorry about that! 😅